src/less/tab/ds6/tab-mixins.less

```diff
@@ -0,0 +1,38 @@
+.tab-outer() {
+    align-items: center;
+    border-bottom: 2px solid transparent;
+    box-sizing: border-box;
+    display: flex;
+    flex: 1;
+    margin: 0 2px;
+    min-height: 40px;
+    -webkit-tap-highlight-color: @ds6-color-g204-gray;
+}
+
+.tab-inner() {
+    color: @ds6-color-text-default;
+    display: flex;
+    flex: 1;
+    font-size: 14px; // font-size should not scale with REM
+    justify-content: center;
+    padding: 8px 16px;
+    text-align: center;
+    text-decoration: none;
+}
+
+.selected-tab-outer() {
+    border-bottom: 2px solid @ds6-color-p002-blue;
+}
+
+.selected-tab-inner() {
+    color: @ds6-color-p002-blue;
+    font-weight: bold;
+}
+
+.hovered-tab-outer() {
+    border-bottom: 2px solid @ds6-color-p002-blue;
+}
+
+.hovered-tab-inner() {
+    color: @ds6-color-p002-blue;
+}
```

Comment:
This mixin doesn't make sense as all the mixins are only used once. Lets normalize them (because the DS4 ones are identical in most cases) and move the rules into the color rules using variables into their respective DS files.

Reply:
I went down that apth, but thought it was clearer this way. Although the mixins are only used once, they  provide clarity. The same mixin names are used across ds4/ds6, but with different mixin definitions, which allows them to share the a lot in `base/tab.less`. I could divide up all the rules to be completely DRY, but I think that the purposes of each section would be muddled with all of them split between several files.

Reply:
**DS4**
```
.tab-outer() {
    align-items: center;
    border-bottom: 2px solid transparent;
    box-sizing: border-box;
    display: flex;
    flex: 1;
    margin: 0 2px;
    min-height: 40px;
    -webkit-tap-highlight-color: @ds4-color-transparent-gray-gainsboro;
}
```
**DS6**
```
.tab-outer() {
    align-items: center;
    border-bottom: 2px solid transparent;
    box-sizing: border-box;
    display: flex;
    flex: 1;
    margin: 0 2px;
    min-height: 40px;
    -webkit-tap-highlight-color: @ds6-color-g204-gray;
}
```

Why can't this go in the base where `.tab-outer()` is used? They are identical except for the last color, and that doesn't need to be a mixin. When DS4 goes away (eventually) we won't need the split files, and the base file will reign.

The pattern we followed for all the other components was not this way, is basically what I'm saying. This will now be different from how we've been doing it.

tl;dr — So, yeah, DRY.

Reply:
>When DS4 goes away (eventually) we won't need the split files, and the base file will reign.

Okay, so we might have a new DS by then. But still ...

Reply:
I see where @yomed is coming from.  Its a little awkward to look across multiple files for declarations that apply to the same selector.  But that's the current pattern everywhere else.  And its DRY, like you said.  If a bug gets fixed in DS6 it could easily get left out of DS4 if we start repeating ourselves.

So I vote for moving these to the base file.